src/state/reducers/companionWindows.js

```diff
@@ -25,6 +25,14 @@ export function companionWindowsReducer(state = {}, action) {
       return removeIn(state, [action.id]);
     case ActionTypes.IMPORT_MIRADOR_STATE:
       return action.state.companionWindows;
+    case ActionTypes.TOGGLE_TOC_NODE:
```

Comment:
Is it possible to add a test for this?

Reply:
I vastly simplified the reducer and added a test.